tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -371,7 +371,9 @@ protected override bool RemoveFast(string key)
             return MaybeUnknown?.Remove(key) ?? false;
         }}
         protected override void ClearFast()
-        {{
+        {{{(loop.ClassName != "FrameRequestHeaders" ? "" : Each(loop.Headers, header => $@"
+            if ({header.TestBit()}) _{header.Identifier} = default(StringValues);"))}
```

Comment:
However with requests, the external-user could stick really long strings in rarely used headers like `Warning` (guessing) and it would hang around if not cleared.